js/app.js

```diff
@@ -1 +1,124 @@
+  anchors.add('h3').remove('.no-anchor');
+}
+
+if (location.search === "?b") {
+  document.querySelector('#main-content > .usa-section:first-child').classList.add('usa-section--dark');
+  document.querySelector('#main-content > .usa-section:nth-child(2)').classList.remove('usa-section--dark');
+  document.querySelectorAll('.bg-primary.radius-pill').forEach(function (el) {
+    el.classList.add('text-white');
+  });
+
+  document.querySelectorAll('img[src*="-on-blue.svg"]').forEach(function (el) {
+    el.src = el.src.replace(/-on-blue/, '-on-white');
+  });
+
+
+  document.querySelectorAll('#main-content > .usa-section:nth-child(2) a[href]:not(.anchorjs-link)').forEach(function (el) {
+    el.classList.add('cg-arrow');
+  });
+} else if (location.search === '?c') {
+  document.querySelector('#main-content > .usa-section:first-child').classList.add('usa-section--dark');
+  document.querySelector('#main-content > .usa-section:nth-child(2)').classList.remove('usa-section--dark');
+  document.querySelector('#main-content > .usa-section:nth-child(2)').classList.add('usa-section--light');
+  document.querySelectorAll('.bg-primary.radius-pill').forEach(function (el) {
+    el.classList.add('text-white');
+  });
+
+  document.querySelectorAll('img[src*="-on-blue.svg"]').forEach(function (el) {
+    el.src = el.src.replace(/-on-blue/, '-on-white');
+  });
+
+  document.querySelectorAll('#main-content > .usa-section:nth-child(2) a[href]:not(.anchorjs-link)').forEach(function (el) {
+    el.classList.add('cg-arrow');
+  });
+
+}
+
+function openCurrentSubNav(self) {
+  self.setAttribute('aria-expanded', !self.getAttribute('aria-expanded'));
+  self.classList.toggle('usa-current');
+  const ariaHandled = self.getAttribute('aria-controls');
+  // //toggle the subnav
+  const controlledElement = document.getElementById(ariaHandled);
+  controlledElement.setAttribute('aria-hidden', (!controlledElement.getAttribute('aria-hidden')).toString());
+  controlledElement.classList.toggle('display-none');
+}
+
+function closeAllLists(listArray) {
+  listArray.forEach(element => {
+    element.setAttribute('aria-hidden', true);
+    element.classList.add('display-none');
+  });
+}
+
+function removeAllUSACurrent(listArray) {
+  listArray.forEach(element => {
+    element.classList.remove('usa-current');
+  });
+}
+
+const sideNavExpansion = function (event) {
+  event.preventDefault();
+  //Close all subnavs
+  if (!this.classList.contains('usa-sidenav__sublist--middle-generation')) {
+    closeAllLists(subLists);
+    removeAllUSACurrent(sideNavParents);
+  } else {
+    closeAllLists(middleSubList);
+    removeAllUSACurrent(sideNavChild);
+  }
+  openCurrentSubNav(this);
+};
+
+for (let i = 0; i < sideNavParents.length; i++) {
+  sideNavParents[i].addEventListener('click', sideNavExpansion, false);
+}
+
+
+if (location.search === "?b") {
+  document.querySelector('#main-content > .usa-section:first-child').classList.add('usa-section--dark');
```

Comment:
is this repeating the earlier dark mode stuff? if so, let's put this in a function that gets called instead of copy/pasted

Reply:
On a diff compare, it's exactly the same.  It can just be removed altogether.  I actually don't know where this got migrated from as it's not in the 11ty template and it's not in cg-site.  If it arises as an issue later, I can address then.